x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -19,41 +63,40 @@ updateStepProgress() {
     --no-progress-meter
 }
 
-echo "Downloading Elastic Agent archive"
+ELASTIC_AGENT_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/beats/elastic-agent/${artifact}.tar.gz"
+echo "Downloading Elastic Agent archive from ${ELASTIC_AGENT_DOWNLOAD_URL}"
 updateStepProgress "ea-download" "loading"
-ELASTIC_AGENT_DOWNLOAD_URL="https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-${ELASTIC_AGENT_VERSION}-linux-x86_64.tar.gz"
 curl -L -O $ELASTIC_AGENT_DOWNLOAD_URL --fail --continue-at -
 if [ "$?" -eq 0 ]; then
   echo "Downloaded Elastic Agent"
   updateStepProgress "ea-download" "complete"
 else
   updateStepProgress "ea-download" "danger"
-  echo "Failed to download Elastic Agent"
-  exit 1
+  fail "Failed to download Elastic Agent"
 fi
 
 echo "Extracting Elastic Agent"
 updateStepProgress "ea-extract" "loading"
-tar -xzf elastic-agent-${ELASTIC_AGENT_VERSION}-linux-x86_64.tar.gz --checkpoint=.1000
+tar -xzf ${artifact}.tar.gz --checkpoint=.1000
```

Comment:
I get a `tar: Option --checkpoint=.1000 is not supported` error on Intel x64

Reply:
removed